### PR TITLE
use interaction data to learn nsrts

### PR DESCRIPTION
**Pull request by tomsilver**
also some refactoring and updates of the score function supercloud script; and copying in @amburger66 's plotting script

### CODE REVIEW

File: src/approaches/interactive_learning_approach.py [hunk 1]
```diff
@@ -305,13 +305,15 @@ def _query_policy(s: State) -> Optional[GroundAtomsHoldQuery]:
...
                 assert isinstance(response, GroundAtomsHoldResponse)
                 state_annotation: List[Set[GroundAtom]] = [set(), set()]
                 for query_atom, atom_holds in response.holds.items():
                     state_annotation[atom_holds].add(query_atom)
-                traj = LowLevelTrajectory([state], [])
-                self._dataset.append(traj, [state_annotation])
+                state_annotations.append(state_annotation)
+            traj = LowLevelTrajectory(result.states, result.actions)
+            self._dataset.append(traj, state_annotations)
         self._relearn_predicates_and_nsrts(
```
Comment (ronuchit) [inline src/approaches/interactive_learning_approach.py:317]: what's going on with these changes? was this a bug before? it looks like you've changed the indentation level at which the traj is getting created

Comment (amburger66) [reply src/approaches/interactive_learning_approach.py:317]: yeah this was kind of a bug, we want the states in each InteractionResult to form a trajectory so this transition data can be used for learning NSRTS, whereas earlier every single state was in its own "trajectory" so there were no transitions for learning nsrts

**Comment by amburger66** [comment]
this lgtm, were there more changes to add?

**Comment by tomsilver** [comment]
I split this code into multiple PRs

**Pull request closed without merging**